core/math/a_star_grid_2d.cpp

```diff
@@ -579,7 +620,7 @@ TypedArray<Dictionary> AStarGrid2D::get_point_data_in_region(const Rect2i &p_reg
 
 			Dictionary dict;
 			dict["id"] = p.id;
-			dict["position"] = p.pos;
+			dict["position"] = _get_solid_unchecked(p.id);
 			dict["solid"] = p.solid;
```

Comment:
So this fails building because `solid` has been removed from the `Point` struct.

Not sure if/how it built previously before the last rebase, but this seems to need more adjustments.